### ISSUE TITLE
Investigation: default date values 

### ISSUE THREAD

**Issue by rssidlowski** [rssidlowski/Pollution_Source_Tracking#172]
From Matt at COB:    Can we have the ability to leave date domains within the investigation page blank?   Some domains that refer to events that may not have occurred yet, such as Solved date, Repair Confirmation Date, Referral Date, and QC Referral Date we would like to default to blank.  Currently these domains are defaulting to today’s date.

I don't see the same behavior. How is this controlled?

**Comment by gerrykelly** [comment]
I only left complaint date being populated

**Comment by gerrykelly** [comment]
but now those other dates have the date-picker come up with 2003 like you had observed before and I had not.  So, I'll take another look at that issue separately,

**Comment by rssidlowski** [comment]
Verified in COBDEV.  Can you provide the ability to clear the date field?  How hard is that?

**Comment by gerrykelly** [comment]
One would think not hard at all.  But Sencha does not support this directly.  However I was able to find some code from a better Sencha programmer than me that gives us not only a clear button on the picker toolbar, but also a "today" button, so that should solve all the date problems.
I also then needed to revise the webservice part so that for each of the Investigation date fields it can actually reset the value to null.  Updated for testing in dev.
